packages/grafana-ui/src/components/Table/StringCell.tsx

```diff
@@ -0,0 +1,47 @@
+import { cx } from '@emotion/css';
+import React from 'react';
+
+import { textUtil } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+import { getCellLinks } from '../../utils';
+import { Button, clearLinkButtonStyles } from '../Button';
+import { DataLinksContextMenu } from '../DataLinks/DataLinksContextMenu';
+
+import { TableCellProps } from './types';
+
+export function StringCell(props: TableCellProps): JSX.Element {
+  const { cell, tableStyles, cellProps, field, row } = props;
+
+  const sanitizeHTML = field.config.custom?.cellOptions.sanitizeHTML;
+  const hasLinks = Boolean(getCellLinks(field, row)?.length);
+  const clearButtonStyle = useStyles2(clearLinkButtonStyles);
+  const content = sanitizeHTML ? (
```

Comment:
😬 -- lets be VERY VERY VERY careful here...  ideally we can just sanitize everything.  if there is a really really good use case for not-sanitized, we need to follow the same pattern as the text panel.   It requires explicit settings in security config to allow this -- and we still get security reports every month telling us how dangerous it is

Reply:
I think it works differently, it looks like the `TextPanel` always sets the content inside `DangerouslySetHtmlContent`, on the string cell when `sanitizeHTML` is off, the cell value gets rendered as text, as React escapes the input by default, only using  `dangerouslySetInnerHTML` when the value is sanitized.